src/content/advanced/storage/azure-pvc-encryption/index.md

```diff
@@ -0,0 +1,78 @@
+---
+linkTitle: Persistent Volumes data encryption with Azure Key Vault
+title: Encrypting Persistent Volumes using Azure Key Vault
+description: Tutorial on how to encrypt dynamically provisioned Persistent Volumes on Azure clusters using Azure Key Vault feature.
+weight: 40
+menu:
+  main:
+    parent: advanced-storage
+user_questions:
+  - How do I encrypt data on Persistent Volumes?
+  - How do I secure data on Persistent Volumes?
```

Comment:
Just in case someone searches for that feature specifically:
```suggestion
  - How do I secure data on Persistent Volumes?
  - How do I use Azure Key Vault?
```